### DISCUSSION TITLE
最新版5.34.0的CalendarPickerView 和CalendarPicker 选择日期组件不能选择年份了吗？而且只展示最近7个月的，以前还能选择年份和任意月份的

### DISCUSSION THREAD

**Discussion by huangmingfu** [ant-design/ant-design-mobile#6539] [Q&A]
![image](https://github.com/ant-design/ant-design-mobile/assets/129028243/f9ff656d-62e1-46ef-8d79-3a63d2d75958)

以前的：
![image](https://github.com/ant-design/ant-design-mobile/assets/129028243/a68d9a9f-878c-4181-af8b-39e8917890c9)

**Comment by zombieJ** [comment]
嗯，新的 Picker 会限定范围。

**Reply by huangmingfu** [reply]
没有一个组件能选择年份了吗？选择年份和月份的需求不是贼多吗？这不是反向升级吗？把原来的那个Calendar组件删掉干嘛

**Reply by zflhcc** [reply]
> 嗯，新的 Picker 会限定范围。

antd-mobile CalendarPicker 设置了 max和min的api   calendarRef.current.jumpTo怎么失效了

**Comment by zflhcc** [comment]
![image](https://github.com/user-attachments/assets/3c01d02e-8734-4567-aad8-61b1b3697cf3)
antd-mobile CalendarPicker 设置了 max和min的api calendarRef.current.jumpTo怎么失效了
打开之后展示的时间不是默认当月，使用jumpTo无效

**Reply by Layouwen** [reply]
有复现demo吗 如：codesandbox
